DDotDDD (#49713)
CleverRaven/Cataclysm-DDA: data/mods/classic_zombies/design-doc.md

The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill will the airborne form of the pandemic even as they tried to fight the zombies.

Within three months, the world was collapsing. Survivors banded together in small groups, now aware of the signs and symptoms of the illness and the danger of a bite. You are one of those survivors. For the last few weeks you've been somewhere safe, holed up, waiting for things to die down, but it's clear now that things aren't going to die down.

### Zombification Mechanics

> The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occuring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.

Reviewer: > Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group.
The group that is susceptible? I am not sure this makes sense.
```suggestion
The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occurring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborne form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.
```

Reviewer 2: I would likewise think the survivors would be clearly NOT in the susceptible group.

(Note that in my head, this infectious agent is too perfect, so it probably has to be some kind of alien invader. However, there's no way the survivors will ever find out, and I'd rather not establish it within game.)

### History
The source of the zombie plague is unclear. About a year ago, rumours arose of a horrible rabies variant arising in Australia and carried by wombats. This strain had jumped to humans and was baffling and frightening doctors. However, those stories died down, and about three months later it was announced to be a novel strain that couldn't transmit significantly in humans.

> Four months ago, new reports of the same effect came up almost simultaneously in Morocco, Belarus, Vietnam, and Portugal. The WHO expressed concern and gave advice for avoiding wild animals. Within a month, the plague had spread to most of the planet. The initial caution about wild animals proved a dangerous red herring, causing people to cluster in urban centers, where the actual vector - other humans and some infected vermin - spread the zombie affliction rapidly. Unfortunately the long latency of the agent meant that it was another month before the ubiquity of the infection became apparent. By the time the second wave of infections became clear, millions of people had been infected, and already passed it to the currently unaware third wave. Hospitals were rapidly overwhelmed, before the revivification risk was even clear. Critically, police and military were among those affected by the unrecognized first wave, and their ranks were as decimated as civilians.

Reviewer 2: ```suggestion
Four months ago, new reports of the same effect came up almost simultaneously in Morocco, Belarus, Vietnam, and Portugal. The WHO expressed concern and gave advice for avoiding wild animals. Within a month, the plague had spread to most of the planet. The initial caution about wild animals proved a dangerous red herring, causing people to cluster in urban centers, where the actual vector - other humans and some infected vermin - spread the zombie affliction rapidly. Unfortunately the long latency of the agent meant that it was another month before the ubiquity of the infection became apparent. By the time the second wave of infections became clear, millions of people had been infected, and already passed it to the currently unaware third wave. Hospitals were rapidly overwhelmed, before the revivification risk was even clear. Critically, police and military were among those affected by the unrecognized first and second waves, and their ranks were as decimated as civilians.
```

> The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill will the airborne form of the pandemic even as they tried to fight the zombies.

Reviewer 2: ```suggestion
The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill with the airborne form of the pandemic even as they tried to fight the zombies.
```

The default CDDA factions do not fit with DDotD without significant change. It is possible that some, like Isherwood, could be rewritten and readded.

#### The Montana Army National Guard
An early faction I'd like to see: the Montana National Guard should have a forward base in Alberta. This group was sent up in the later part of the apocalypse to aid Canada's self-defense. Within a few days, they lost contact with command. They have chosen to establish a reinforced base, and are fighting off zombies and attempting to rally survivors. They may, at some point, change their name once their makeup is less than half American. They have been here for weeks and are running low on supplies.

#### The Central Alberta Historical Re-Enactors Army
CAHREA was a local club of creative anachronists. By some coincidence, a large number of their members were immune to the initial zombie infection. Their access to armour that is additionally protective against zombie bites gave them an edge, and they began helping evacuate and protect survivors in a rural community center. Although the original CAHREA group is now a small minority, they have maintained the name.

#### The Wild Rose Survivors

> This is a group of rural Albertans less hard-hit by the pandemic portion of the plague due to their remote and spaced-out living. They were still eventually affected as the plague became widespread. They tend to be well equipped with pickup trucks and hunting rifles, somewhat insular, but welcoming - if a bit suspicoius - of other survivors.

Reviewer 2: ```suggestion
This is a group of rural Albertans less hard-hit by the pandemic portion of the plague due to their remote and spaced-out living. They were still eventually affected as the plague became widespread. They tend to be well equipped with pickup trucks and hunting rifles, somewhat insular, but welcoming - if a bit suspicious - of other survivors.
```
